src/vs/platform/telemetry/common/commonProperties.ts

```diff
@@ -23,13 +23,16 @@ export function resolveCommonProperties(
 	commit: string | undefined,
 	version: string | undefined,
 	machineId: string | undefined,
+	sqmId: string | undefined,
 	isInternalTelemetry: boolean,
 	product?: string
 ): ICommonProperties {
 	const result: ICommonProperties = Object.create(null);
 
 	// __GDPR__COMMON__ "common.machineId" : { "endPoint": "MacAddressHash", "classification": "EndUserPseudonymizedInformation", "purpose": "FeatureInsight" }
 	result['common.machineId'] = machineId;
+	// __GDPR__COMMON__ "common.sqmId" : { "endPoint": "MacAddressHash", "classification": "EndUserPseudonymizedInformation", "purpose": "BusinessInsight" }
```

Comment:
MacAddressHash isn't the right endpoint. It's `SQMMachineId`

Reply:
Good catch will fix.